server/src/libs/constants/api.constants.js

```diff
@@ -0,0 +1,9 @@
+import { config } from '../packages/config/config.js';
+import { ApiPath, AuthApiPath } from '../enums/enums.js';
+
+const WHITE_ROUTES = [
```

Comment:
You are using WHITE_ROUTES only into /libs/packages/server-application/server-application